backend/tracim_backend/tests/__init__.py

```diff
@@ -137,21 +152,25 @@ class FunctionalTestEmptyDB(FunctionalTest):
 class FunctionalTestNoDB(FunctionalTest):
     config_section = 'functional_test_no_db'
 
+    def override_settings(self, settings):
+        settings['sqlalchemy.url'] = 'sqlite://'
```

Comment:
because this test is a special one "FunctionalTestNoDB" . This test when database url is uncorrect, idea is to override here to avoid complexity when rewritting sqlalchemy.url with sed in travis.